### PR TITLE
docs: release announcement for 1.22

### PR DESCRIPTION
**Pull request by gbartolini**

### CODE REVIEW

File: content/blog/cloudnative-pg-1-22.0-released/index.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+**CloudNativePG Operator**, a significant milestone featuring the introduction
+of **declarative tablespaces** and **temporary tablespaces** alongside various
+enhancements and fixes.
+
+Tablespaces, a powerful and widely adopted feature in database management
+systems, now take center stage in CloudNativePG 1.22.0. This release empowers
+users to boost the vertical scalability of their databases by decoupling the
+physical and logical data modeling, achieving optimal performance through
+parallel on-disk read/write operations. With CloudNativePG, users can define
+additional tablespace volumes, and also ensure they are used to temporary
```
Comment (jsilvela) [inline content/blog/cloudnative-pg-1-22.0-released/index.md:29]:
```suggestion
additional tablespace volumes, and also ensure they can be used for temporary
```

File: content/blog/cloudnative-pg-1-22.0-released/index.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+systems that enhances the vertical scalability of a database, by decoupling the
+physical and logical modeling of data and by optimizing performance through
+parallel on-disk read/write operations. CloudNativePG enables you to define
+additional tablespace volumes, and control whether they can be used for
+temporary operations (by seamlessly managing the `temp_tablespaces` GUC).
+
+Tablespaces are another important step towards Very Large Database adoption,
+following the introduction of Kubernetes volume snapshots in 1.21.
+
+From 1.22, the `ALTER SYSTEM` command will be disabled by default, to ensure
```
Comment (jsilvela) [inline content/blog/cloudnative-pg-1-22.0-released/index.md:82]: this and following ... it's a duplicate. @gbartolini are you experimenting different forms, or is this a copy-paste artifact?

**Review by jsilvela — CHANGES_REQUESTED** [review]
we have a dupe there I'd like to understand.
Will also keep this one blocked until D-day.

File: content/blog/cloudnative-pg-1-22.0-released/index.md (hunk 2 above)
Comment (jsilvela) [reply content/blog/cloudnative-pg-1-22.0-released/index.md]: Mostly dupe. I added a bit of this one to the "first copy".
Will approve this PR but not merge it or publish it, leaving that to you @gbartolini .
The release documents are now published.

**Review by jsilvela — APPROVED** [review]

**Review by mnencia — APPROVED** [review]

**Review by leonardoce — APPROVED** [review]

**Review by jsilvela — APPROVED** [review]

**Pull request merged**